contracts/Pricing.sol

```diff
@@ -111,12 +111,12 @@ contract Pricing is IPricing {
                     } else {
                         staleHour = 23;
                     }
-                    updatePrice(0, 0, 0, true, staleHour);
+                    createPriceEntry(0, 0, 0, staleHour);
                 }
             }
 
             // update time of last hourly recording
-            startLastHour = block.timestamp;
+            startLastHour += (elapsedHours * 3600);
```

Comment:
```suggestion
            startLastHour += (elapsedHours * 1 hours);
```

(I think this works)

Reply:
Not sure if this is makes it clearer. I personally think the constant 3600 makes it clearer that a conversion between hours to seconds is occurring.

Reply:
Is it possible to do the following?
```
            startLastHour += elapsedHours hours;
```

Reply:
Just tried, does not work